docs/src/modules/components/DiamondSponsors.js

```diff
@@ -33,16 +34,38 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function DiamondSponsors() {
+export default function DiamondSponsors(props) {
   const classes = useStyles();
+  const { spot } = props;
   const t = useSelector((state) => state.options.t);
 
   return (
     <div className={classes.root}>
-      <Typography variant="caption" color="textSecondary" display="block">
+      <Typography variant="caption" color="textSecondary" display="block" gutterBottom>
         {t('diamondSponsors')}
       </Typography>
       <a
+        data-ga-event-category="sponsor"
+        data-ga-event-action={spot}
+        data-ga-event-label="sencha"
+        href="https://www.sencha.com/products/extreact/extreact-for-material-ui/?utm_source=materialui&utm_medium=referral&utm_content=product-200429-extreactmaterialui"
+        rel="noopener noreferrer sponsored"
+        target="_blank"
+        style={{ marginLeft: 8, width: 125, height: 35 }}
+      >
+        <NoSsr>
```

Comment:
Why?

Reply:
Chrome displays the following while the image is loading

<img width="292" alt="Capture d’écran 2020-05-02 à 22 12 27" src="https://user-images.githubusercontent.com/3165635/80891080-05f97500-8cc2-11ea-96c0-487516ce0164.png">

I couldn't find a better solution

Reply:
That is how the web displays alternative text for images. This is how it's supposed to behave.

Please remove the NoSsr so that the browser has more opportunities to prioritize resources and we can run less JS.

Reply:
Regarding why it behaves like this, my assumption is that the browser quickly realizes that he doesn't have the network capability to fetch and display the image before a few seconds. He decides to display the alt text as a fallback.

Now, in our case, I think that we would be better off without this fallback, removing one state transition. From 3 states: transparent -> alt text (that looks like a broken image) -> image loaded to 2 states: transparent -> image loaded, which is less distraction.

---

I would be happy to replace the NoSsr component with a better alternative. I'm looking for an alternative to https://amp.dev/documentation/examples/components/amp-img/.

```jsx
<amp-img
  noloading
  src="my-source.svg"
  width="208"
  height="40"
  alt="AMP by Example"
/>
```

I have used their `display: inline-block` to avoid layout jumps when the image is loading. 

 What about we leverage this hook?

https://github.com/mui-org/material-ui/blob/126cea6d2f7bd396bed636eb3a8c132e17012616/packages/material-ui/src/Avatar/Avatar.js#L59

I have added more information on this item: https://trello.com/c/rVLCjWOk/1910-image-component, in case, we want to prioritize a module for the problem in the future.